qutebrowser/utils/standarddir.py

```diff
@@ -301,7 +301,7 @@ def _from_args(
         suffix = basedir_suffix[typ]
     except KeyError:  # pragma: no cover
         return None
-    return os.path.abspath(os.path.join(args.basedir, suffix))
```

Comment:
I've located the problem here, apparently pathlib.resolve doesn't behave the same way on windows for non existing path [https://stackoverflow.com/questions/42513056/how-to-get-absolute-path-of-a-pathlib-path-object](url) (though I haven't checked in a vm)

I'm going to eat but there are still some changes that I need to push for this one